### ISSUE TITLE
Optimization of PEtab problem fails with TypeError

### ISSUE THREAD

**Issue by IvaEwert** [ICB-DCM/pyPESTO#537]
Good evening, 

I am using the latest pyPESTO develop branch and tried parameter estimation with two models including the Zhao2020 model from the benchmark collection.
Some time ago my script worked like this for me, but now I get the following error message:
```
Traceback (most recent call last):
  File "/Zhao_QuantBiol2020/example_parameter_estimation_Zhao2020.py", line 25, in <module>
    result = optimize.minimize(problem=problem, optimizer=optimizer,
  File "/pypesto/pypesto/optimize/optimize.py", line 87, in minimize
    startpoints = assign_startpoints(
  File "/pypesto/pypesto/startpoint/util.py", line 49, in assign_startpoints
    x_sampled = startpoint_method(
TypeError: startpoint_method() missing 1 required positional argument: 'n_samples'
```
Attached you can find the PEtab files and the example script. Do you have any idea what might be the problem here?

Thank you in advance!
[example.zip](https://github.com/ICB-DCM/pyPESTO/files/5654049/example.zip)

**Comment by dweindl** [comment]
> Do you have any idea what might be the problem here?

Yes, https://github.com/ICB-DCM/pyPESTO/pull/535#discussion_r536370229

**Comment by jvanhoefer** [comment]
Thank you for pointing this out. Is fixed in #538
